src/Core/Services/OrganizationService.cs

```diff
@@ -1,19 +1,28 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
```

Comment:
I don't see this using being used, if that's the case remove it (use the automatic clear of using)